src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Repository/ServiceRepository.php

```diff
@@ -119,4 +119,10 @@ public function findByTeamNames($teamNames)
             'teamName' => $teamNames,
         ]);
     }
+
+    public function delete(Service $service)
```

Comment:
Missing typehint

Reply:
The type hint is present, I did not add a doc block on purpose as the type hint is explicit enough in my opinion.